index.html

```diff
@@ -1881,28 +1850,22 @@
           <li>If <a>connection list promise</a> is not <code>null</code>,
           return <a>connection list promise</a> and abort these steps.
           </li>
-          <li>Otherwise, set <a>connection list promise</a> be a new
+          <li>Otherwise, let <a>connection list promise</a> be a new
```

Comment:
We use <em>P</em> to hold an unsettled promise elsewhere in the spec, can we do the same here?